### PR TITLE
master -> main

### LINKED ISSUE #1178: Rename default branch from `master` to `main`
# Feature Request

## Description
Terra's default branch is named 'master'. While it's a term that we rarely think about, the term 'master' is racially loaded. Such terminology has no place in our inclusive development culture and we will not perpetuate their use.

With this issue we will rename our default branch to `main` and update our CI builds and rules.

## Additional Context / Screenshots
https://www.zdnet.com/article/github-to-replace-master-with-alternative-term-to-avoid-slavery-references/
https://www.cnet.com/news/twitter-engineers-replace-racially-loaded-tech-terms-like-master-slave/

### PR DESCRIPTION
**Pull request by mjhenkes**
### Summary
Update references from master to main
Closes #1178

### Deployment Link

https://terra-framework-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

**Comment by mjhenkes** [comment]
No other PRs can be merged until this one is. The build is currently broken. I'm going to merge #1166 into this PR and get the build green again.

File: packages/terra-application-navigation/src/tabs/_TabRollup.jsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jsx-a11y/role-supports-aria-props */
```
Comment (mjhenkes) [inline packages/terra-application-navigation/src/tabs/_TabRollup.jsx:1]: Logged to investigate: https://github.com/cerner/terra-framework/issues/1181

File: packages/terra-navigation-side-menu/src/_MenuItem.jsx [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jsx-a11y/role-supports-aria-props */
```
Comment (mjhenkes) [inline packages/terra-navigation-side-menu/src/_MenuItem.jsx:1]: Addressed here: #1164

File: packages/terra-application-navigation/src/tabs/_TabRollup.jsx (hunk 1 above)
Comment (mjhenkes) [reply packages/terra-application-navigation/src/tabs/_TabRollup.jsx:1]: actually also address here too: #1164

File: packages/terra-application-utility/CHANGELOG.md [hunk 3]
```diff
@@ -31,7 +34,7 @@ Unreleased
 * Added clinical-lowlight-theme wdio screenshots
 
 2.31.0 - (February 25, 2020)
-------------------
+------------------ß
```
Comment (benbcai) [inline packages/terra-application-utility/CHANGELOG.md:37]:
```suggestion
------------------
```

**Pull request merged**